src/picongpu/include/simulation_defines/param/speciesAttributes.param

```diff
@@ -140,6 +140,11 @@ alias(atomicNumbers);
  */
 alias(effectiveNuclearCharge);
 
+/*! alias for particle population kinetics model, e.g. FLYlite
```

Comment:
I'll keep it that way since all attributes below and above have the same formatting error. we can update them all together in a follow-up